lib/cdo/github.rb

```diff
@@ -39,6 +39,26 @@ def self.merge_pull_request(pr_number)
     response['merged']
   end
 
+  # Creates and merges a pull request.
+  # @param base [String] The base branch of the requested pull request.
+  # @param head [String] The head branch of the requested pull request.
+  # @param title [String] The title of the requested pull request.
+  # @return [nil | Integer] The PR number of the newly created DTT if successful
+  #   or nil if unsuccessful.
+  def self.create_and_merge_pull_request(base:, head:, title:)
+    pr_number = create_pull_request(base: base, head: head, title: title)
+    success = merge_pull_request(pr_number)
+    success ? pr_number : nil
+  end
+
+  # Builds the HTML URL from a pull request number. Does not validate the pull
+  # request number.
+  # @param pr_number [Integer] The pull request number.
+  # @return [String] The HTML URL for the pull request.
+  def self.get_html_url(pr_number)
```

Comment:
Debated this myself. The response from `Octokit.create_pull_request` contains a number of URLs, including the `html_url` (their name) as well as `url` (e.g., https://api.github.com/repos/code-dot-org/code-dot-org/pulls/13653), `diff_url` (e.g., https://github.com/code-dot-org/code-dot-org/pull/13653.diff), `issue_url` (e.g., https://api.github.com/repos/code-dot-org/code-dot-org/issues/13653), and others.

Depending on how explicit we want to be, it may or may not make sense to drop the `html`.